src/migrations/M00010.js

```diff
@@ -0,0 +1,26 @@
+// This migration removes references to dependent devices from
+// the database, as we are no longer pursuing dependent devices.
+export async function up(knex) {
+	// Remove dependent key from each image
+	if (await knex.schema.hasColumn('image', 'dependent')) {
```

Comment:
I don't think you need to test that the column and table exist, the migrations are ran in order, so you can assume that the structure of migration 9 is what exists before this migration. It doesn't really hurt if you want to leave it though